README.md

```diff
@@ -14,10 +15,25 @@ This module is dedicated for Python 3.x users
 Current version (0.1.0) is not available on PyPi, although once a <br>
 stable version is ready it will be pushed to PyPi repo.<br><br>
 
-For now, installation can be done from this GIT repository, using <br>
+
+As you may see in requirements.txt file beprof uses both numpy and scipy.<br>
+To make installing them possible, you might need to download some packages<br>
+which you can do (on UNIX) with:
+`~$ sudo apt-get install -y gfortran libopenblas-dev liblapack-dev`
+
+For now, installation can be done: 
+* via pip
+From this GIT repository, using <br>
 `~$ pip3 install git+https://github.com/grzanka/beprof.git@master` <br>
+(where `@master` refers to the name of a branch)<br>
 
-(where `@master` refers to the name of a branch)
+* manually
+Providing that you have already install necessery packages<br>
+```
+git clone https://github.com/grzanka/beprof.git
+cd beprof
+python setup.py install
```

Comment:
C:\Users\User\PycharmProjects\beprof>python setup.py install --user
C:\Users\User\Anaconda3\lib\distutils\dist.py:260: UserWarning: Unknown distribution option: 'install_requires'

I have numpy 1.9.4 and in my case install_requires is ignored.